.env.template

```diff
@@ -2,14 +2,10 @@
 FLASK_APP=todo_app/app
 FLASK_ENV=development
 
-# Change the following values for local development.
-SECRET_KEY=secret-key
-MONGODB_COLLECTIONNAME=
+
+PROD_GITHUB_CLIENT_ID=
+PROD_GITHUB_CLIENT_SECRET=
```

Comment:
Does having the word `PROD` in here mean anything? Surely your development environment will also be setting this variable with `PROD`, as will any other environment?